improver/spotdata/apply_lapse_rate.py

```diff
@@ -31,15 +31,19 @@
 
 """Apply temperature lapse rate adjustments to a spot data cube."""
 
+import warnings
+
 import iris
 import numpy as np
 from iris.cube import Cube
+from iris.exceptions import CoordinateNotFoundError
 
-from improver import PostProcessingPlugin
+from improver import BasePlugin
+from improver.metadata.probabilistic import is_probability
 from improver.spotdata.spot_extraction import SpotExtraction, check_grid_match
 
 
-class SpotLapseRateAdjust(PostProcessingPlugin):
+class SpotLapseRateAdjust(BasePlugin):
```

Comment:
Changing the class from `PostProcessingPlugin` to `BasePlugin` has caused the output cube attribute `title` to change so that it no longer starts with the string `Post-Processed `. I cannot see a reason for making this change.

Reply:
This change was in order to fix a bug we found that overwrites the user specified title. I have updated the information in the description on this PR now to highlight this point.